src/org/etools/j1939_84/controllers/StepController.java

```diff
@@ -278,15 +278,14 @@ protected void compareRequestPackets(List<? extends GenericPacket> globalPackets
         }
     }
 
-    protected void checkForNACKs(List<? extends GenericPacket> globalPackets,
-                                 List<? extends AcknowledgmentPacket> dsAcks,
-                                 String section) {
+    protected void checkForNACKsDS(List<? extends GenericPacket> dsPackets,
```

Comment:
This should be named `globalPacket`.  This method should be named `checkForNACKsGlobal`.

Additionally, `checkForNACKsFromObdModules()` should be renamed to `checkForNACKsDS`